templates/telemetry/crash/crash.4.schema.json

```diff
@@ -121,10 +125,6 @@
               "description": "<percentage>, Windows-only, percent of memory in use",
               "type": "string"
             },
-            "TelemetrySessionId": {
-              "description": "<id>, Active telemetry session ID when the crash was recorded",
-              "type": "string"
-            },
```

Comment:
This has already made it into `generated-schemas` so this field can't be removed. If this appears in additional properties but is camelCased instead, it would be fine to change it here, otherwise I would update the description to mention that this is an obsolete field as of bug 1624986.

Reply:
I don't think the problem is that it's obsolete so much as it was never correct in the first place. But I have no problem leaving it in!